package.json

```diff
@@ -34,12 +34,13 @@
     "postinstall": "npx yarn-deduplicate"
   },
   "devDependencies": {
+    "@babel/plugin-transform-react-jsx-self": "^7.12.1",
+    "@babel/plugin-transform-react-jsx-source": "^7.12.1",
     "@babel/preset-env": "^7.11.0",
     "@babel/preset-react": "^7.10.4",
     "@babel/preset-typescript": "^7.10.4",
     "@compiled/jest": "*",
     "@compiled/react": "*",
-    "@emotion/core": "^10.1.1",
```

Comment:
removed the emotion test because i'd need to upgrade it to v11 for the new automatic stuff. rather just remove.